### ISSUE TITLE
TextField validation in Stack widget asking for Dropdown options

### ISSUE THREAD

**Issue by GuzSantos** [FlutterFlow/flutterflow-issues#2356]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I have build a page with 5 textfields and organize them under a Stack widget. The fields are to enter decimal numbers. When I enter an action to validate the Form, it shows a message error for select option from Dropdown even if the number is available. On settings is asking to set 2 message errors.

When I place a textfield outside the stack widget the Form validation only ask for one message error
![Screen Shot 2024-02-17 at 11 28 55](https://github.com/FlutterFlow/flutterflow-issues/assets/139913617/235bad7d-ec36-465a-9c6f-6c12e851244a)

see the Form validation setup. First option TextNeck: 2 message errors

### Expected Behavior

The form validation should validate only one message error like the TextFieldOutsideStack

### Steps to Reproduce

1. create a page
2. add a column
3. add a stack
4. add a TextField inside the Stack widget
5. add a TextField outside the Stack widget
6. Wrap the page with Form validation
7. Add a FAB button with the action to validate the form
8. Run the test mode

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT48zfLl8Y92scsA1aqBbO9VliYWHDo7aucv1O1+UU08KOvZPO0Me/WkPxRtZMejTQlmJUSmmkUDwNLSvoflV/I7PQ+WRqpazrVUUw7gJUOte5OBDqqWfUFDJ99PGky70cKz2SJAKtBuSEIEw2WMIN7Ka3TrK5jtIlFISq/LZO4=

### Context

I have an Onboarding page where I need to validate if the information entered is correct. When the user add the correct information the double message error is blocking them to proceed.

### Visual documentation

![Screen Shot 2024-02-17 at 11 28 55](https://github.com/FlutterFlow/flutterflow-issues/assets/139913617/cf201507-a12d-48d7-8785-c38303a2505c)
![Screen Shot 2024-02-17 at 11 35 33](https://github.com/FlutterFlow/flutterflow-issues/assets/139913617/cd7d4ac5-26a6-4850-87ac-4065278dd4bd)

### Environment

```markdown
- FlutterFlow version: v4.1
- Platform: Web
- Browser name and version: Chrome 121.0.6167.160
- Operating system and version affected: macOS Catalina 10.5
```

**Comment by GuzSantos** [comment]
I need someone to review it urgently. The page is part of my onboarding process and I can't add a user without the TextField validation

**Comment by hariprasadms** [comment]
Hi @GuzSantos - Thank you for bringing up the issue. I've verified the blank project and reviewed the issue setup you provided. Your details have been incredibly helpful. Upon debugging, I've observed validation messages appearing for all text fields as expected.

Could you please clarify your expectations regarding this issue?

<img width="402" alt="image" src="https://github.com/FlutterFlow/flutterflow-issues/assets/17435678/4937a239-b331-40bc-94ed-f7870c09fb23">

**Comment by hariprasadms** [comment]
Hey @GuzSantos, Looks like not more activity in the issue, so I'm guessing the issue has been sorted out on your end. If you still think there's a problem, feel free to start a new issue with all the latest details and screenshots if needed.

**Comment by GuzSantos** [comment]
Hi Hari,I am disappointed. No one reach out to check the issue. There a 2 validation error when should be just one. If is not a bug, please tell me how to remove the message to get an option from dropdown.ThanksGustavo Enviado do meu iPhoneEm 28 de fev. de 2024, à(s) 11:20, Hari ***@***.***> escreveu:﻿
Closed #2356 as not planned.
